i18n/lt.toml

```diff
@@ -89,8 +89,8 @@ pvExport = "Tinklo eksportas"
 
 [main.mode]
 off = "Stop"
-minpv = "Min+Saulės"
-pv = "Saulės"
+minpv = "Min+Saulė"
+pv = "Saulė"
```

Comment:
There are still a lot of unrelated translations in this PR.

Reply:
I did these a couple days ago on Weblate..

Reply:
Then this should be rebased